types/messaging.ts

```diff
@@ -270,6 +270,13 @@ interface ShowFirefoxAccounts {
   data: {
     /** Adds entrypoint={your value} to the FXA URL */
     entrypoint: string;
```

Comment:
Now that I look at the implementation these should both be optional, no? i.e. `entrypoint?` and `extraParams?`